source/StartConditions.cpp

```diff
@@ -106,3 +177,19 @@ const list<Ship> &StartConditions::Ships() const
 {
 	return ships;
 }
+
+
+
+bool StartConditions::IsValid() const
+{
+	if(system && (!planet || !system->IsValid()))
+		return false;
+	
+	if(planet && (!system || !planet->IsValid() || planet->GetSystem() != system))
+		return false;
+	
+	if(!stockConversation && conversation.IsEmpty())
+		return false;
```

Comment:
The system & planet checks are overly complex. Note that you'll need to merge in latest `master` for `Conversation::IsValidIntro` to compile.
```suggestion
	// A start must specify a valid system.
	if(!system || !system->IsValid())
		return false;
	
	// A start must specify a valid planet in its specified system.
	if(!planet || !planet->IsValid() || planet->GetSystem() != system)
		return false;
	
	// A start must reference a valid "intro" conversation, either stock or custom.
	if((stockConversation && !stockConversation->IsValidIntro()) || (!stockConversation && !conversation.IsValidIntro()))
		return false;
```